ibm/resource_ibm_cm_offering_instance.go

```diff
@@ -93,6 +98,24 @@ func resourceIBMCmOfferingInstance() *schema.Resource {
 				Optional:    true,
 				Description: "id of the resource group",
 			},
+			"install_plan": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				Default:     "Automatic",
```

Comment:
Is it `Automatic` by default earlier as well? 
Since it is Optional and Default.. it might effect existing users.. Is that fine?
Can you test this scenario.. Try creating offering instance using `1.26.2` and try to do `terraform apply` again with this dev binary

Reply:
Yes automatic has always been by default, but that's happened behind the scenes. This is a new capability to allow for manual install plans

Reply:
On second thought though, I will remove the default, because we won't want to set that for instances when we expand support beyond operators.

Reply:
thanks